### DISCUSSION TITLE
能否透露新版的大概发布时间？

### DISCUSSION THREAD

**Discussion by lingyezhixing** [OleehyO/TexTeller#50] [Q&A] [answered]
很抱歉打扰您的开发。最近临近高校期末，simple tex使用人数激增，经常识别失败，能否透露一下新版的大概更新时间？如果还有好久才能更新我就准备去用付费的识别软件了。

**Comment by OleehyO** [comment, marked as answer]
新版本大概在6月10号之前发布，现在测试下来已经不输于SimpleTex了，可以稍微等待一下👀
